doc/source/user_guide/hpc/launch_mapdl_entrypoint.rst

```diff
@@ -0,0 +1,235 @@
+
+.. code-block:: python
+
+    mapdl.exit()
+
+When the Python process you are running PyMAPDL on finishes without errors, and you have not
+issued :meth:`exit() <ansys.mapdl.core.Mapdl.exit>`, the garbage collector
+kills the MAPDL instance and its job. This is intended to save resources.
+
+If you prefer that the job is not killed, set the following attribute in the
+:class:`Mapdl <ansys.mapdl.core.mapdl.MapdlBase>` class:
+
+.. code-block:: python
+
+    mapdl.finish_job_on_exit = False
+
+
+In that case, it is recommended you set a timeout in your job to avoid having the job
+running more than needed.
+
+
+Handling crashes on an HPC
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If MAPDL crashes while running on HPC, the job finishes right away.
+In that case, MAPDL disconnect from MAPDL.
+PyMAPDL retries to reconnect to the MAPDL instance up to 5 times waiting
+for up to 5 seconds.
+If unsuccessful, you might get an error like this:
+
+.. code-block:: text
+
+    MAPDL server connection terminated unexpectedly while running:
+    /INQUIRE,,DIRECTORY,,
+    called by:
+    _send_command
+
+    Suggestions:
+    MAPDL *might* have died because it executed a not-allowed command or ran out of memory.
+    Check the MAPDL command output for more details.
+    Open an issue on GitHub if you need assistance: https://github.com/ansys/pymapdl/issues
+    Error:
+    failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:50052: Failed to connect to remote host: connect: Connection refused (111)
+    Full error:
+    <_InactiveRpcError of RPC that terminated with:
+    status = StatusCode.UNAVAILABLE
+    details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:50052: Failed to connect to remote host: connect: Connection refused (111)"
+    debug_error_string = "UNKNOWN:Error received from peer  {created_time:"2024-10-24T08:25:04.054559811+00:00", grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:127.0.0.1:50052: Failed to connect to remote host: connect: Connection refused (111)"}"
+    >
+
+The data of that job is available at :attr:`directory <ansys.mapdl.core.Mapdl.directory>`.
+It is recommended you set the run location using ``run_location`` argument.
+
+While handling this exception, PyMAPDL also cancels the job to avoid resources leaking.
+Therefore, the only option is to start a new instance by launching a new job using
+:func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>`.
+
+User case on a SLURM cluster
+----------------------------
+
+Assume an user wants to start a remote MAPDL instance in a HPC cluster
+to interact with it.
+The user would like to request 10 nodes, and 1 task per node (to avoid clashes
+between MAPDL instances). 
+He would like to also request 64 GB of memory RAM.
+Because of administration logistic, he must use the machines in
+`supercluster01` partition.
```

Comment:
```suggestion
Assume a user wants to start a remote MAPDL instance in an HPC cluster
to interact with it.
The user would like to request 10 nodes, and 1 task per node (to avoid clashes
between MAPDL instances). 
The user would like to also request 64 GB of RAM.
Because of administration logistics, the user must use the machines in
the ``supercluster01`` partition.
```